packages/dashboard/src/karma/fixture.js

```diff
@@ -17,7 +17,7 @@
 /**
  * External dependencies
  */
-import * as React from 'react';
+import * as React from '@web-stories-wp/react';
```

Comment:
This doesn't work. It makes this fail in karma tests. 

https://github.com/google/web-stories-wp/blob/547bba0dd7f8e3ce76eafa6b88406fcc1ba7d68e/packages/dashboard/src/karma/fixture.js#L104-L121

Any idea why this might be the case @swissspidy ?

Reply:
Hmm might be a difference in how `@web-stories-wp/react` has `"type": "module"`, but the `react` package is all CommonJS.

And Karma/Jasmine cannot spy on module exports (but can for CJS packages), causing the `createElement is not declared writable or has no setter` message.

It's something I've encountered myself in #5792 too, where I am looking into possible workarounds.

I recommend keeping `import * as React from 'react';` for now and leave this for this separate PR to figure out.